lms/djangoapps/certificates/views/webview.py

```diff
@@ -249,9 +249,11 @@ def _update_course_context(request, context, course, course_key, platform_name):
                                                               '{partner_short_name}.').format(
             partner_short_name=context['organization_short_name'],
             platform_name=platform_name)
-    # If language specific templates are enabled for the course, add course_run specific information to the context
-    if CertificateGenerationCourseSetting.is_language_specific_templates_enabled_for_course(course_key):
-        fields = ['start', 'end', 'max_effort', 'content_language']
+    course_certificate_settings = CertificateGenerationCourseSetting.get(course_key)
+    if course_certificate_settings and course_certificate_settings.language_specific_templates_enabled:
+        fields = ['content_language']
+        if course_certificate_settings and course_certificate_settings.include_hours_of_effort:
```

Comment:
You don't need the first half of this check, since you already know from line 253 that `course_certificate_settings` exists.